### DISCUSSION TITLE
GNOME UI Theme?

### DISCUSSION THREAD

**Discussion by Deewens** [Floorp-Projects/Floorp#1418] [Q&A] [answered]
Hi,

I installed Floorp, and on the setup page, it told me that if I wanted to have a specific theme that fit with my desktop I could go to the settings. However, there is no theme setting about any desktop environment. So, I had a look on Floorp release note page, and found out that you recently removed GNOME UI support. I would like to understand why you've done that, and if there is any 3rd party alternatives that I can use? As I fell really used to the GNOME theme, chrome.css on barebone Firefox.

Thanks!

**Comment by CutterKnife** [comment, marked as answer]
https://blog.ablaze.one/4364/2024-06-14/
The themes other than Proton and Lepton have caused many problems, but were removed with the upgrade to ESR128 because there were not enough resources to fix them. However, the Fluerial design has been reworked into something completely new.
If they can afford it, a new design maybe added in the future. (Or contributions are welcome!)

**Reply by Deewens** [reply]
Thanks!
